niworkflows/interfaces/surf.py

```diff
@@ -725,3 +725,14 @@ def get_gii_meta(in_file):
         in_file = in_file[0]
     gii = load(in_file)
     return gii.darrays[0].meta.metadata
+
+
+def get_crown_cifti(dilated_brainmask):
+    """Extract the surface from the dilated brain mask
```

Comment:
Currently CIFTI doesn't include WM+CSF, so I think what we need for carpetplot is:

* Functional file in boldref space
* Gray matter segmentation or CIFTI
* Crown (or brain) mask
* WM/CSF mask (together or separate)
* aCompCor mask

I think with this reorganization, this function is not going to be needed. I'm not sure how you would get a surface here, anyway.

Reply:
Probably, the best way here is:
1. Dilate the brain mask as above, but not as far as the dilation to calculate the outer rim of the circular crown.
2. Run marching cubes (e.g., FreeSurfer's implementation) to obtain a surface.
3. Probably, smooth and severely decimate.